src/components/Analysis/Modes/LoadedModesList.spec.js

```diff
@@ -19,7 +19,12 @@ jest.mock('react-router-dom', () => ({
   useParams: jest.fn().mockReturnValue(() => ({ purpose: '', mode: '' }))
 }))
 jest.mock('src/components/Analysis/helpers')
-jest.mock('src/lib/timeseries')
+jest.mock('src/lib/timeseries', () => ({
+  computeAggregatedTimeseries: jest.fn(),
+  sortTimeseriesByCO2GroupedByMode: jest.fn(),
+  computeCO2Timeseries: jest.fn(),
+  transformTimeseriesToTrips: jest.fn(() => [])
```

Comment:
to test the sorted feature, we should return something inside this table, that would be inverted by the added sort
```
transformTimeseriesToTrips: jest.fn(() => [])
```

Reply:
pourquoi est-ce qu'on a besoin de tout mocker ici ?

Reply:
Le fichier était déjà mock avant `jest.mock('src/lib/timeseries')`
J'aurais pu laisser tel quel ce mock et ajouter `transformTimeseriesToTrips.mockReturnValue([])` effectivement 👍